nursery/get-windows-directory-from-kuser_shared_data.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: get Windows directory from KUSER_SHARED_DATA
+    namespace: host-interaction/file-system
+    authors:
+      - david.cannings@pwc.com
+    scope: basic block
+    references:
+      - http://www.rohitab.com/discuss/topic/42325-the-kuser-shared-data-structure/
+      - https://www.geoffchappell.com/studies/windows/km/ntoskrnl/inc/api/ntexapi_x/kuser_shared_data/index.htm
+    examples:
+      - 7f15b1a47bbe031334e23653879e9661f4b8cde80c307548328fdd3aed87ca46:0x01019080
+  features:
+    # Fixed offset of Windows directory path in KUSER_SHARED_DATA structure.
+    - number: 0x7ffe0030 = KUSER_SHARED_DATA.NtSystemRoot
```

Comment:
is this offset the same for both 32 and 64 bit windows?

Reply:
> KUSER_SHARED_DATA is a data structure that contains a lot of Windows system information. It is always located in user mode at 0x7ffe0000, even on 64-bit Windows. At 0x7ffe0030, the full path of the Windows directory is stored there. The Windows version number is stored at 0x7ffe026c and 0x7ffe0270. 0x7ffe026c is the major version number, and 0x7ffe0270 is the minor version number.

http://www.rohitab.com/discuss/topic/42325-the-kuser-shared-data-structure/

well, ok then!

given that this is an obvious question, i'd recommend a short comment indicating that we've already checked and confirmed. otherwise, looks good!